deploy/deploy-latest-image.sh

```diff
@@ -0,0 +1,128 @@
+#Fail fast
+set -e
+
+# ACR
+ACR_NAME="spektateacr"
```

Comment:
Do you think we should consider pushing to the devcrewsacr image for Spektate, which is picked up by the MCR image?

Reply:
@samiyaakhtar does that mean every merge to master will be picked up by MCR?

Reply:
It would push to it but it won't be the one being pulled down by the MCR reference in spk. We need to introduce a tag, let's say prod, to point to the most stable release of spektate and tag that one as let's say `prod` in MCR and make sure spk dashboard command pulls that one. And then we would go in weekly and tag a release that has been verified as prod in MCR.  

But we can keep pushing versioned tags to this image as well, such as tags based on dates. Thoughts? I'm reading [here](https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375) on how using the latest tag is not safe and we should introduce versioning.

Reply:
Yes, only using the latest tag is not recommended. We can have a build tag and set the latest build tag to a latest tag and re-push